rep-2001.rst

```diff
@@ -45,6 +45,8 @@ We define various entry points for ROS users.
  * `perception`: The `perception` variant includes packages commonly used for perception capabilities.
  * `simulation`: The `simulation` variant includes packages that are needed for automated simulation,
    without extra debugging tools.
+ * `ros-build-essential`: The `ros-build-essential` ROS distribution independent variant includes only the packages that are required for building and compiling ROS 2 packages.
```

Comment:
I think the problem with this statement is that a user who is unfamiliar with the distinction might think this is enough to compile locally.  So I might reword this to use the wording we are talking about elsewhere, something like:

```suggestion
 * `ros-build-essential`: The `ros-build-essential` ROS distribution independent variant includes only the packages that are required for building ROS 2 packages on the buildfarm.
```

Reply:
I dislike "on the buildfarm" because it's not entirely correct. It _is_ possible to build ROS packages with just these, but it isn't easy and there's additional effort involved but it's probably more succinct to just equate the build farm with that situation.

However I cannot bring myself to merge an incomplete truth so here's a counter-suggestion:

```suggestion
 * `ros-build-essential`: The `ros-build-essential` ROS distribution independent variant includes only the packages that are required for building standalone ROS 2 packages. ROS developers should use `ros-dev-tools` instead for the full set of recommended tools.
```

Reply:
Your update works for me, thanks.